sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_hub_operation.py

```diff
@@ -98,8 +94,8 @@ def get(self, name: str, **kwargs: Dict) -> WorkspaceHub:
 
         return workspace_hub
 
-    # @monitor_with_activity(logger, "Hub.BeginCreate", ActivityType.PUBLICAPI)
     @distributed_trace
+    @monitor_with_activity(logger, "WorkspaceHub.BeginCreate", ActivityType.PUBLICAPI)
```

Comment:
Was it intentional to re-activated logging for the workspace hub methods in this PR?

Reply:
Not sure on this one, I just enabled it because it seemed unintended. Checking with appropriate teammate now. @ZhidaLiu

Reply:
this was always turned off because we were not clear on what it is doing. I believe it is emitting some metrics to SDK team, right? unless there is some concern or error for turning it on, then our team has no preference. please let me know if you think it should be enabled or disabled @diondrapeck

Reply:
Yes, it's used for logging to appinsights. There's no concern or error.